pyomo/contrib/mindtpy/iterate.py

```diff
@@ -22,15 +25,20 @@ def MindtPy_iteration_loop(solve_data, config):
             % solve_data.mip_iter)
 
         if algorithm_should_terminate(solve_data, config, check_cycling=False):
+            cuts_add_last_iteration = True
             break
 
         solve_data.mip_subiter = 0
         # solve MILP master problem
-        if config.strategy == 'OA':
+        if config.strategy in {'OA', 'GOA'}:
```

Comment:
This is just a question. Why not using a list of string? Something like `['OA', 'GOA']`?